swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
 class Item:
-    pass
\ No newline at end of file
+    
+    def __init__(self, category=str(), condition=0, age=0): #using empty string constructor to explicitly note string data type expected
+        self.category = category
+        self.condition = condition
+        self.age = age
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        if self.condition == 0:
+            return "barely sellable"
+        elif self.condition == 1:
+            return "heavily used"
+        elif self.condition == 2:
+            return "meh"
+        elif self.condition == 3:
+            return "reasonable"
+        elif self.condition == 4:
+            return "good"
+        elif self.condition == 5:
+            return "mint"
```

Comment:
Nice handling for condition description! It looks like the code assumes that `self.condition` will always be an integer, but what if we allowed decimals like a condition of `3.5`? How could we adapt our code to convert the decimal to an int we could use here, or alternately, change the if-statements to support decimals?

Another approach for finding the condition description could be to make a dictionary that maps conditions as keys to descriptions as values. There's a trade off of using more memory creating the map variable, but if memory isn't a concern, for me, it's a little easier to read.
```suggestion
        condition_map = {
            0: "barely sellable",
            1: "heavily used",
            2: "meh",
            3: "reasonable",
            4: "good",
            5: "mint"
        }
        
        return condition_map[int(self.condition)]
```

Reply:
This makes sense! I really like how this approach allows for a single return statement.